src/commonmark/en/content/user/using-the-maps-app.md

```diff
@@ -1371,7 +1371,8 @@ write is visible to those people.
 2.  Click **Interpretations** in the top right of the workspace to open
     the interpretations panel.
 
-3.  Click + to write a new interpretations.
+3.  A textfield will appear with a placeholder "Write an interpretation" for users
```

Comment:
"textfield" should be "text field" for _normal_ users

Reply:
Done! I've split up the word